source/release/4.2.4/index.html.md

```diff
@@ -0,0 +1,251 @@
+---
+title: oVirt 4.2.4 Release Notes
+category: documentation
+layout: toc
+---
+
+# oVirt 4.2.4 Release Notes
+
+The oVirt Project is pleased to announce the availability of the 4.2.4 First Release Candidate as of May 24, 2018.
+
+oVirt is an open source alternative to VMware™ vSphere™, providing an
+awesome KVM management interface for multi-node virtualization.
+This release is available now for Red Hat Enterprise Linux 7.4,
```

Comment:
Shouldn't this be 7.5 already?

Reply:
Update the script that generates the notes here: https://gerrit.ovirt.org/91641